meson_options.txt

```diff
@@ -1,3 +1,12 @@
+# Fully qualified application ID for the emulator
+# See https://dbus.freedesktop.org/doc/dbus-specification.html#message-protocol-names for details
+option(
+    'app_id',
+    type: 'string',
+    value: 'org.dosbox-staging.Dosbox-staging',
```

Comment:
Is it a requirement that the last .dot value have 'Name' case (i.e, 'Dosbox'?)

Because DOSBox Staging uses both that spelling as well as 'dosbox-staging' for filenames (similar to how DOSBox upstream was named).

(It's never been 'Dosbox', so it would be nice to set this to: `'org.dosbox-staging.dosbox-staging'`)

Reply:
From the FreeDesktop specs:
> Interface names should start with the reversed DNS domain name of the author of the interface (in lower-case), like interface names in Java. It is conventional for the rest of the interface name to consist of words run together, with initial capital letters on all words ("CamelCase")

This isn't a strict requirement. I switched it to `org.dosbox-staging.DOSBoxStaging`, does that work?

Reply:
org.dosbox-staging.DOSBoxStaging -> Probably nothing wrong with that, but if the pattern is `<organisation>.<project>` then the organiation and repo here on GitHub is probably an even better match:

- `https://github.com/<organization>/<project-repo-name>`
- `https://github.com/dosbox-staging/dosbox-staging`

Also, the project website is https://www.dosbox-staging.org/, so that's another 1-to-1 match. 

So my vote would be org.dosbox-staging.dosbox-staging :slightly_smiling_face: 

Will wait for long term team members to weigh in!

Reply:
> So my vote would be `org.dosbox-staging.dosbox-staging` 🙂

I second this. I think being clever in URLs with non-lowercase and non-ASCII chars is inviting trouble, regardless of what is technically allowed. And as @interloper98 put it, this maps nicely to our website URL.

Reply:
Well, or just `org.dosbox-staging` as I said in my other comment. We won't start adding other projects with the `org.dosbox-staging` prefix, you can take that for granted 😆